### DISCUSSION TITLE
Why is a GitHub repo's insights chart of git clone number decreasing?

### DISCUSSION THREAD

**Discussion by PrimeTheFirst** [community/community#22371] [New to GitHub]
<h2>
<a name="why-is-a-github-repos-insights-chart-of-git-clone-number-decreasing-1" class="anchor" href="#why-is-a-github-repos-insights-chart-of-git-clone-number-decreasing-1"></a>Why is a GitHub repo’s insights chart of git clone number decreasing?</h2>
<p>First the value was 402. Then 204(the next day) . The day after that 102. Now it’s 71 .</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>That number is supposed to be the number of clones that happened <em>on that day</em>. Varying activity can depend on all sorts of things, for example changing interest by people or even the number of CI runs that clone the repository.</p>

**Comment by PrimeTheFirst** [comment]
<p>Ah , ok . I thought it was the total number of clones altogether. Thanks a ton !</p>
